specification/2.0/README.md

```diff
@@ -3900,67 +3888,144 @@ Application-specific data.
 
-As previously defined
+```
+metal_brdf =
+  microfacet_brdf(alpha) * fresnel(HdotV, baseColor) +
+  multiscatter_microfacet_brdf(alpha) * multiscatter_fresnel(baseColor)
+```
 
-`const dielectricSpecular = rgb(0.04, 0.04, 0.04)`
-<br>
-`const black = rgb(0, 0, 0)`
+The microfacet BRDF uses the GGX distribution and Smith height-correlated masking-shadowing term ([Heitz 2014: Understanding the Masking-Shadowing Function in Microfacet-Based BRDFs](http://jcgt.org/published/0003/02/03/paper.pdf)).
 
-*c<sub>diff</sub>* = `lerp(baseColor.rgb * (1 - dielectricSpecular.r), black, metallic)`
-<br>
-*F<sub>0</sub>* = `lerp(dieletricSpecular, baseColor.rgb, metallic)`
-<br>
-*&alpha;* = `roughness ^ 2`
+```
+     1            alpha^2
+D = -- --------------------------------
+    pi   (HdotN^2 * (alpha^2-1) + 1)^2
 
-Additionally,  
-*V* is the normalized vector from the shading location to the eye  
-*L* is the normalized vector from the shading location to the light  
-*N* is the surface normal in the same space as the above values  
-*H* is the half vector, where *H* = normalize(*L*+*V*)  
+                                           2 * NdotV * NdotL
+G = ----------------------------------------------------------------------------------------------
+    NdotL * sqrt(NdotV^2 * (1-alpha^2) + alpha^2) + NdotV * sqrt(NdotL^2 * (1-alpha^2) + alpha^2))
 
-The core lighting equation the sample uses is the Schlick BRDF model from [An Inexpensive BRDF Model for Physically-based Rendering](https://www.cs.virginia.edu/~jdl/bib/appearance/analytic%20models/schlick94b.pdf)
 
-![](figures/lightingSum.PNG)
+                         G * D
+microfacet_brdf = --------------------
+                    4 * NdotL * NdotV
+```
 
-Below are common implementations for the various terms found in the lighting equation.
+The Schlick Fresnel is computed via the reflection half vector `H = normalize(V + L)`.
 
-### Surface Reflection Ratio (F)
+```
+fresnel = baseColor + (1 - baseColor) * (1 - HdotV)^5
+```
 
-**Fresnel Schlick**
+The multi-scattering approximation uses the directional albedo `E_m` and average albedo `E_mavg` of the single-scattering model. These values are typically precomputed and fetched from a lookup table during rendering. The average Fresnel `F_mavg` for Schlick can be computed analytically.
```

Comment:
This is a bit of a stumbling point for me.  Should we link to [Enterprise PBR Shading Model](https://dassaultsystemes-technology.github.io/EnterprisePBRShadingModel/spec-2021x.md.html#appendix/energycompensation) for background? Can we provide these precomputed lookup tables?  I see that your github provides these here:
[GGX_E](https://github.com/DassaultSystemes-Technology/EnterprisePBRShadingModel/blob/master/res/GGX_E.exr)
[GGX_E_avg](https://github.com/DassaultSystemes-Technology/EnterprisePBRShadingModel/blob/master/res/GGX_E_avg.exr)

It might also be worth mentioning the numerical fit described in [Energy Preservation](https://dassaultsystemes-technology.github.io/EnterprisePBRShadingModel/spec-2021x.md.html#components/core/in-depthlookatthebsdfs) for those who are willing to sacrifice energy conservation error in exchange for one less texture lookup into floating point texture lookups.

Reply:
Worth noting that I came across this when trying to see how a new gltf material would be written, using this appendix as reference.  I really appreciate the well defined abstractions and diagrams that have been developed thus far; looking forward to seeing the final writeup!

Reply:
Not sure if we should link to specific implementation in the spec, is it possible to do that? I was also thinking about describing how to build the tables (similar to [this)](https://dassaultsystemes-technology.github.io/EnterprisePBRShadingModel/spec-2021x.md.html#appendix/energycompensation), but maybe that is too much detail and we should link to the [slides from Imageworks](https://blog.selfshadow.com/publications/s2017-shading-course/imageworks/s2017_pbs_imageworks_slides_v2.pdf) or [other implementation-independent sources](https://blog.selfshadow.com/2018/05/13/multi-faceted-part-1/). We could also put the tables into the glTF repo.